src/main/java/org/zalando/nakadi/validation/schema/PartitionStrategyConstraint.java

```diff
@@ -0,0 +1,17 @@
+package org.zalando.nakadi.validation.schema;
+
+import org.zalando.nakadi.domain.EventType;
+import org.zalando.nakadi.domain.EventTypeBase;
+
+import java.util.Optional;
+
+public class PartitionStrategyConstraint implements SchemaEvolutionConstraint {
```

Comment:
All these constraints are related to event type properties related to the schema. I couldn't think about a better name. Related to the this comment as well https://github.com/zalando/nakadi/pull/487#discussion_r89524324